packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -145,155 +145,239 @@ describe('when only one permission matches', () => {
               },
             ],
-            demandedPermissions: ['ViewProducts'],
-            demandedActionRights: [
-              { group: 'products', name: 'PublishProducts' },
-            ],
-            shouldMatchSomePermissions: true,
+            demandedPermissions: ['ViewCustomers'],
           });
 
-          expect(queryByText('Is authorized: No')).toBeInTheDocument();
+          expect(
+            rendered.queryByText('Is authorized: Yes')
+          ).toBeInTheDocument();
         });
       });
-      describe('if can edit prices on products group', () => {
-        it('should pass isAuthorized as "false"', () => {
-          const { queryByText } = testRender({
+      describe('when demanding manage permission', () => {
+        it('should indicate being authorized', () => {
+          const rendered = render({
             allAppliedPermissions: [
-              { name: 'canManageProjectSettings', value: true },
-              { name: 'canViewProducts', value: true },
-            ],
-            allAppliedActionRights: [
               {
-                group: 'products',
-                name: 'canEditPrices',
+                name: 'canManageCustomers',
                 value: true,
               },
             ],
-            demandedPermissions: ['ViewProducts'],
-            demandedActionRights: [{ group: 'products', name: 'EditPrices' }],
-            shouldMatchSomePermissions: true,
+            demandedPermissions: ['ManageCustomers'],
           });
 
-          expect(queryByText('Is authorized: Yes')).toBeInTheDocument();
+          expect(
+            rendered.queryByText('Is authorized: Yes')
+          ).toBeInTheDocument();
         });
       });
-      describe('if can edit prices on orders group', () => {
-        it('should pass isAuthorized as "false"', () => {
-          const { queryByText } = testRender({
-            allAppliedPermissions: [
-              { name: 'canManageProjectSettings', value: true },
-              { name: 'canViewProducts', value: true },
-            ],
-            allAppliedActionRights: [
-              {
-                group: 'orders',
-                name: 'canEditPrices',
-                value: true,
-              },
-            ],
-            demandedPermissions: ['ViewProducts'],
-            demandedActionRights: [{ group: 'products', name: 'EditPrices' }],
-            shouldMatchSomePermissions: true,
-          });
+    });
+    describe('without applied manage permission', () => {
+      it('should indicate not being authorized', () => {
+        const rendered = render({
+          allAppliedPermissions: [
+            {
+              name: 'canManageCustomers',
+              value: true,
+            },
+          ],
+          demandedPermissions: ['ViewCustomersGroups'],
+        });
+
+        expect(rendered.queryByText('Is authorized: No')).toBeInTheDocument();
+      });
+    });
+  });
+});
+
+describe('action rights', () => {
```

Comment:
Wondering if we should write the describes using the ACs. Would make the specs easier to read maybe? 🤔

Reply:
Yeah I felt the urge too at some point. However, didn’t want to mix changing code and tests too much in a single PR. If we decide to to that I would take that on on another PR so things are easier to trace and this isn’t the „when they changed too much PR and things broke“ 🤪🧚🏻

Reply:
ye if that makes things easier for you, you can split this into a separate step